src/main/java/org/checkerframework/checker/linear/LinearAnalysis.java

```diff
@@ -106,10 +106,10 @@ protected boolean canUpdate(Node lhsNode, Node rhsNode, CFValue lhsValue) {
                 atypeFactory.getAnnotationMirror(rhsNode.getTree(), Unique.class);
         if (lhsAMUnique != null && rhsAMUnique != null && lhsValue != null) {
             for (AnnotationMirror lhsAnno : lhsValue.getAnnotations()) {
-                if (AnnotationUtils.areSameByName(atypeFactory.UNIQUE, lhsAnno)) {
+                if (AnnotationUtils.areSameByName(linearAtypeFactory.UNIQUE, lhsAnno)) {
                     List<String> lhsStatesList =
                             AnnotationUtils.getElementValueArray(
```

Comment:
The deprecated version that takes the executable name allows getting default value, by passing `true` as last argument.
The suggested replacement only takes a default value, leading to duplication between the qualifier declaration and all these uses.
This use doesn't provide a default, which can fail at runtime.
I've filed https://github.com/eisop/checker-framework/issues/1064 to clean this up.